microsoft-365-dke/README.md

```diff
@@ -3,28 +3,99 @@ https://docs.microsoft.com/en-us/microsoft-365/compliance/double-key-encryption
 
 # Prerequisites
 
-1. Make sure you have permissions to access Microsoft azure portal, and also can click the create new app service and create new app registration buttons. link: https://portal.azure.com/ 
+1. Make sure you can access [Microsoft Azure portal](https://portal.azure.com/), and have the following permmisions:
+    
+    a. Create new app service.
 
-2. Make sure you have permission to access Microsoft 365 compliance and you can click the create new label button : https://compliance.microsoft.com/informationprotection?viewid=sensitivitylabels 
+    b. Create new app registration.
 
-3. make sure you have Microsoft 365 E5 license.
+2. Make sure you can access access [Microsoft 365 compliance](https://compliance.microsoft.com/informationprotection?viewid=sensitivitylabels) and you have the following permmisions:
+
+    a. Create new label.
+    
+    b. Publish a label.  
+
+3. Make sure you have Microsoft 365 E5 license.
 
 4. From UKC server:
- a. create a partition "test"
- b. create an RSA key in partition "test" with the following command : ucl generate -t RSA -n <key_name> -p test.
+
+    a. Create a partition name "test".
```

Comment:
@mdvir the partition name does not have to be 'test' right - and an existing partition can be used so I think this should be described more accurately